lib/fs.js

```diff
@@ -2069,10 +2098,10 @@ function WriteStream(path, options) {
 
   if (this.start !== undefined) {
     if (typeof this.start !== 'number') {
-      throw new TypeError('"start" option must be a Number');
+      throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'start', this.start);
     }
     if (this.start < 0) {
-      throw new Error('"start" must be >= zero');
+      throw new errors.Error('ERR_INVALID_OPT_VALUE', 'start', this.start);
```

Comment:
Similar to L1944

Reply:
I updated it to `ERR_VALUE_OUT_OF_RANGE`